x/blob/types/blob_tx.go

```diff
@@ -102,12 +103,12 @@ func ValidateBlobTx(txcfg client.TxEncodingConfig, bTx *tx.BlobTx, subtreeRootTh
 	}
 
 	// verify that the commitment of the blob matches that of the msgPFB
+	calculatedCommitments, err := inclusion.CreateParallelCommitments(bTx.Blobs, merkle.HashFromByteSlices, subtreeRootThreshold, runtime.NumCPU()*4)
```

Comment:
> runtime.NumCPU()*4

I think this is too much and will just have go routines fighting for resources. let's set it to runtime.NumCPU()

Reply:
Yeah, this number was kinda random, just wanted to see your measurements and then adjust

Reply:
which measurements? you mean I should run the changes to see the performance improvements?